lighthouse-core/lib/i18n/i18n.js

```diff
@@ -71,6 +71,11 @@ const formats = {
       minimumFractionDigits: 1,
       maximumFractionDigits: 1,
     },
+    percent4SigFig: {
+      // Force allow up to two digits after decimal place in percentages. (Intl.NumberFormat options)
+      maximumFractionDigits: 4,
```

Comment:
wouldn't this be 2 then? maximumSignificantDigits is its own thing

still would want to keep this at one though :)

Reply:
Since it takes the percent decimal in the form `0.XXXX` -> `XX.XX%` it has to be 4 decimal places to prune it to 2 after the decimal place.  I think maximumSignificantDigits would also effectively do the same thing?

Reply:
Oh I was very confused then

how does this get turned back into `67.X%`? `style: 'percent'` in ICU automatically multiplies by 100?

seems like we should either use `maximumSignificantDigits` and keep the name or change the name and keep the `maximumFractionDigits` with extra-emphasis that NumberFormat is bonkers and applies the maximumFractionDigits to the pre-multiplied string

I apologize for my confusion, I think I'd prefer sigFig because that's just nuts 😆

Reply:
Yes `style: 'percent'` will multiply by 100 and add a `%` to the number automatically and round off so there are no digits after the decimal place.  I can try to use sigfigs, the only worry i have is that setting 4 sigfigs might lead to `8.546%` which would be accurately 4 sigfigs, but not really what we want, I'd imagine we want `8.55%` to be consistent.